graphql_compiler/tests/snapshot_tests/test_cost_estimation.py

```diff
@@ -617,13 +660,9 @@ def test_optional_then_filter(self):
             'Event_RelatedEvent': 23,
             'Event': 7,
             'FeedingEvent': 6,
+            'BirthEvent': 13
         }
-        statistics = LocalStatistics(
-            class_counts=count_data,
-            edge_count_between_vertex_pairs=dict(),
-            count_of_distinct_field_values=dict(),
-            histograms=dict()
-        )
+        statistics = LocalStatistics(count_data)
```

Comment:
Can you split out the changes that remove all the empty `dict()` parameters into a separate PR? It will shave hundreds of lines that are not that relevant or difficult to review, and make it easier for us to focus on the actual changes of substance here.